docs/ambassador.md

```diff
@@ -0,0 +1,32 @@
+
+# Ambassador
+
+The Ambassador API Gateway provides all the functionality of a traditional ingress controller
+(e.g., path-based routing) while exposing many additional capabilities such as authentication,
+URL rewriting, CORS, rate limiting, and automatic metrics collection.
+
+## Ambassador Operator
+
+This Ambassador addon deploys the Ambassador Operator, which in turn will install Ambassador in
+a kubespray cluster.
```

Comment:
In a kubernetes cluster ? I'm not really sure we can call it a "kubespray cluster" as we are only bootstraping/scaffholding a k8s installation